src/Features/VisualBasic/Portable/ExtractMethod/VisualBasicMethodExtractor.vb

```diff
@@ -70,8 +71,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.ExtractMethod
             Return VisualBasicCodeGenerator.GenerateResultAsync(insertionPoint, selectionResult, analyzeResult, cancellationToken)
         End Function
 
-        Protected Overrides Function GetFormattingRules(document As Document) As IEnumerable(Of AbstractFormattingRule)
-            Return SpecializedCollections.SingletonEnumerable(Of AbstractFormattingRule)(New FormattingRule()).Concat(Formatter.GetDefaultFormattingRules(document))
+        Protected Overrides Function GetFormattingRules(document As Document) As ImmutableArray(Of AbstractFormattingRule)
+            Return ImmutableArray.Create(Of AbstractFormattingRule)(New FormattingRule()).AddRange(Formatter.GetDefaultFormattingRules(document))
```

Comment:
small nit.  but i'd love if we could pull up the Formatter.GetDfeaultFormattingRules to the base and just have the derived types only responsible for creating and returning their one specialied rule that goes first.

Reply:
➡️ Consolidated this